src/ingest_validation_tools/directory-schemas/codex.yaml

```
@@ -1,65 +1,72 @@
-# A dataset consists of a single CODEX experimental run, and is
-# collected by the Akoya software into a single directory.
-#
-# directory beginning with drv_ contains analyzed files produced.
-# There are a few files in this directory that are used for the HuBMAP
-# data processing pipelines (channelNames.txt, experiment.json, segmentation.json)
-#
-# directory beginning with src_ contains the raw (unprocessed by Akoya) files.
+# A dataset consists of a single CODEX experimental run
 
 -
   pattern: '[^/]*NAV[^/]*\.tif'
   description: 'Navigational Image showing Region of Interest (Keyance Microscope only)'
   required: False
 -
   example: 'summary.pdf'
-  pattern: '.+\.pdf'
+  pattern: '[^/].*\.pdf'
   description: 'PDF export of Powerpoint slide deck containing the Image Analysis Report'
   is_qa_qc: True
+  required: False
 -
-  pattern: 'drv_[^/]+/channelNames\.txt'
-  description: 'Text file produced by the Akoya software which contains the (linearized) channel number and the Name/ID/Target of the channel (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)/.*'
+  description: 'Raw data directory.'
   required: True
 -
-  pattern: 'src_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(processed|drv_.*)/.*'
```

```suggestion
  pattern: '(processed|drv_[^/]*)/.*'
```
Suggest that matches for file or directory names use `[^/]` consistently. Will not comment everywhere this occurs.